botan-bindings/botan-bindings.cabal

```diff
@@ -14,61 +14,74 @@ source-repository head
 
 library
     hs-source-dirs:   src
+
     default-language: Haskell2010
-    default-extensions:
-        NoImplicitPrelude
-        OverloadedStrings
-        PatternSynonyms
-        ScopedTypeVariables
-    exposed-modules:
-        Botan.Bindings.Bcrypt
-        Botan.Bindings.BlockCipher
-        Botan.Bindings.Cipher
-        Botan.Bindings.Error
-        Botan.Bindings.FPE
-        Botan.Bindings.Hash
-        Botan.Bindings.HOTP
-        Botan.Bindings.KDF
-        Botan.Bindings.KeyWrap
-        Botan.Bindings.MAC
-        Botan.Bindings.MPI
-        -- Botan.Bindings.PBKDF
```

Comment:
Are these commented out modules still coming later? @ldillinger

Because these commented out lines have now been removed.

Reply:
`botan_pbkdf` and `botan_pbkdf_timed` have been deprecated in favor of `botan_pwdhash` and `botan_pwdhash_timed` respectively.

`kyber` is a key encapsulation method, `scrypt` is a password-based key derivation function. Neither have specialized FFI functions, so general interfaces must be used for them.